src/components/ui/SuggestsDropdown.jsx

```diff
@@ -11,37 +11,29 @@ const SuggestsDropdown = ({ className = '', suggestItems, onSelect, onHighlight
     const keyDownHandler = e => {
       const { key } = e;
 
-      if (key === 'ArrowDown') {
-        const h = highlighted === null ? -1 : highlighted;
-
-        if (h < suggestItems.length - 1) {
-          setHighlighted(h + 1);
-          onHighlight(suggestItems[h + 1]);
-        } else {
-          setHighlighted(null);
-          onHighlight(null);
+      if (key === 'Enter') {
+        if (highlighted !== null) {
+          e.preventDefault(); // prevent search input submit with its current content (highlighted POI name)
+          onSelect(highlighted);
         }
       }
 
-      if (key === 'ArrowUp') {
-        e.preventDefault(); // prevent cursor returning at beggining
-        const h = highlighted === null ? suggestItems.length : highlighted;
+      if (key !== 'ArrowDown' && key !== 'ArrowUp') {
+        return;
+      }
 
-        if (h > 0) {
-          setHighlighted(h - 1);
-          onHighlight(suggestItems[h - 1]);
-        } else {
-          setHighlighted(null);
-          onHighlight(null);
-        }
+      let newHighlighted;
+      if (key === 'ArrowDown') {
+        newHighlighted = suggestItems[suggestItems.indexOf(highlighted) + 1] || null;
       }
 
-      if (key === 'Enter') {
-        if (highlighted !== null) {
-          e.preventDefault(); // prevent search input submit with its current content (highlighted POI name)
-          onSelect(suggestItems[highlighted]);
-        }
+      if (key === 'ArrowUp') {
+        e.preventDefault(); // prevent cursor returning at beggining
+        newHighlighted = suggestItems[suggestItems.indexOf(highlighted) - 1] || null;
       }
```

Comment:
It's no longer possible to cycle up through the list with  :arrow_up:  'ArrowUp'. Is it intentional ?

The current behavior was specified for consistency with Qwant Search, and is actually useful to access easily to the favorites that may appear at the end of the suggestions.

Reply:
Thank you, that was a mistake, I fixed it.